docs/effects/api.md

```diff
@@ -50,42 +45,13 @@ dispatched by effect streams.
 Filter actions by action type. Accepts multiple action types.
 
 Usage:
-```
-actions$.ofType('LOGIN', 'LOGOUT');
-```
-
-
-## Effect
 
-Decorator that marks a class property or method as an effect. Causes the
-decorated observable to be subscribed to `Store` when the effect class is
-ran.
-
-Usage:
 ```ts
-class MyEffects {
-  constructor(private actions$: Actions, private auth: AuthService) { }
-
-  @Effect() login$: Observable<Action> = this.actions$
-    .ofType('LOGIN')
-    .switchMap(action =>
-      this.auth.login(action.payload)
-        .map(res => ({ type: 'LOGIN_SUCCESS', payload: res }))
-        .catch(err => Observable.of({ type: 'LOGIN_FAILURE', payload: err }))
-    );
-
-  @Effect() logout(): Observable<Action> {
-    return this.actions$
-      .ofType('LOGOUT')
-      .switchMap(() =>
-        this.auth.logout()
-          .map(res => ({ type: 'LOGOUT_SUCCESS', payload: res }))
-          .catch(err => Observable.of({ type: 'LOGOUT_FAILURE', payload: err }))
-      );
-  }
-}
+actions$.ofType('LOGIN', 'LOGOUT');
 ```
 
+
+### Terminating Effects
```

Comment:
IMO, "Terminating Effects" sounds like a way to kill an effect from running completely rather than meaning it won't automatically dispatch to the store. Maybe my terminology is just incorrect though.